src/migrations/2023080309391234-update-goal-on-approved-ar-value.js

```diff
@@ -0,0 +1,32 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+
+      await queryInterface.sequelize.query(`
+        UPDATE "Goals"
```

Comment:
This is only fixing the false positives, should the false negatives also be checked?

Reply:
Let me know if your seeing something different.